src/_premake_init.lua

```diff
@@ -285,6 +285,10 @@
 			"Default",
 			"GDB",
 			"LLDB",
+			"Local",
+			"Remote",
```

Comment:
I don't like `Local` & `Remote` here, I think those states are implied by `debugremotehost` being set or not no?